cypress/e2e/06_add_building/add_building.cy.js

```diff
@@ -0,0 +1,565 @@
+describe('Going through the Add Building Workflow', function () {
+
+    beforeEach(() => {
+        cy.login();
+        cy.visit('/plugins/init-workflow');
+    });
+
+    it('Go through the workflow and populate all fields', function () {
+        cy.contains('Workflows');
+        cy.contains('Add Building').click();
+        cy.wait(2000);
+
+        // Add Building 1st page
+        cy.get('[aria-label="Start new or please select from below"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').contains('HA/01').click();
+        cy.get('.btn-success').contains('Start New').click();
+        cy.wait(2000);
+
+        // Start tab
+        cy.wait(4000);
+        cy.get('.card_component.resourceid').contains('HA Number');
+        cy.get('.form-control').should('be.disabled');
+        cy.get('.btn-success').contains('Save and Continue').click();
+
+        // Heritage Asset Details tab
+        cy.contains('Heritage Asset Name');
+        cy.get('[aria-label="Heritage Asset Name"]').click().type('Testing');
+
+        cy.contains('Heritage Asset Type');
+        cy.get('[aria-label="Heritage Asset Type"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
+
+        cy.contains('Historical Period').scrollIntoView();
+        cy.get('[aria-label="Historical Period"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
+        cy.wait(4000);
+
+        cy.get('.bibliographic_source_citation').contains('Bibliographic Source').scrollIntoView();
+        cy.wait(2000)
+         cy.get('.bibliographic_source_citation').contains('Add new Relationship').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').first().click();
```

Comment:
What if this doesnt exist? This should be created in the general tests and accessed by name